support/cas-server-support-pac4j-core-clients/src/test/java/org/apereo/cas/support/pac4j/web/flow/DummySingleLogoutService.java

```diff
@@ -0,0 +1,185 @@
+package org.apereo.cas.support.pac4j.web.flow;
+
+import java.util.List;
+import java.util.Set;
+
+import javax.xml.namespace.QName;
+
+import org.opensaml.core.xml.Namespace;
+import org.opensaml.core.xml.NamespaceManager;
+import org.opensaml.core.xml.XMLObject;
+import org.opensaml.core.xml.schema.XSBooleanValue;
+import org.opensaml.core.xml.util.AttributeMap;
+import org.opensaml.core.xml.util.IDIndex;
+import org.opensaml.saml.saml2.metadata.SingleLogoutService;
+import org.w3c.dom.Element;
+
+import net.shibboleth.utilities.java.support.collection.LockableClassToInstanceMultiMap;
+
+
+/**
+ * Dummy implementation of {@link SingleLogoutService} just for unit tests.
+ * 
+ * Does not do anything.
+ * 
+ * @author jkacer
+ * 
+ * @since 5.3.0
+ */
+public class DummySingleLogoutService implements SingleLogoutService {
```

Comment:
Remove this. Use mocks please.

Reply:
You are right, this can be replaced by a mock easily. Done.